README.md

```diff
@@ -45,10 +45,10 @@ tableView.render(box)
 
 #### Sections and Nodes 🏗
 
-A `Section` and a `Node` are building blocks of the `Box`:
+`Section`s and `Node`s are building blocks of a `Box`:
 
-- The `Section` is an abstraction of `UITableView`'s section, which defines whether there is going to be any header or footer.
-- The `Node` is an abstraction of `UITableView`'s row, it defines how it's going be rendered.
+- `Section` is an abstraction of `UITableView`'s section which defines whether `header`s or `footer` should be shown.
```

Comment:
Should probably be just `header`

Reply:
`which defines whether a `header` or `footer` should be shown.`

Reply:
fixed https://github.com/Babylonpartners/Bento/pull/24/commits/32b15609a9a55a842e79cc722ce389a0284653f7